gomanifest/internal/deps.go

```diff
@@ -28,68 +29,54 @@ type DepPackage struct {
 	Deps       []string  `json:"Deps"`
 }
 
-// GoListCmd ... Go list command structure.
-type GoListCmd struct {
-	CWD string
-}
-
-// Run ... Actual function that executes go list command and returns output as string.
-func (goListCmd GoListCmd) Run() (io.ReadCloser, error) {
-	GoListGoListDeps := exec.Command("go", "list", "-json", "-deps", "./...")
-	GoListGoListDeps.Dir = goListCmd.CWD
-	defer GoListGoListDeps.Start()
-	GoListGoListDeps.Wait()
-	return GoListGoListDeps.StdoutPipe()
-}
-
 // GoListCmdInterface ... Interface to be implemented to execute go list command.
-type GoListCmdInterface interface {
-	Run() (io.ReadCloser, error)
-}
-
-// GoList ... Structure that handle go list data and extract required packages.
-type GoList struct {
-	Command GoListCmdInterface
+type GoList interface {
```

Comment:
This interface belong to golist, can be moved to golist.go file.

Reply:
golist.go is an implementation of GoList interface. Usually generic types should have concrete dependency.